gradle.properties

```diff
@@ -1,22 +0,0 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
-pockethub_github_client=dummy_client
-pockethub_github_secret=dummy_secret
-pockethub_github_callback=http://dummy.example.com
```

Comment:
We shouldn't change the content of this file with our credentials, but use the gradle.properties file in the Gradle user home directory:

> You can place a gradle.properties file in the Gradle user home directory (defined by the `GRADLE_USER_HOME` environment variable, which if not set defaults to `USER_HOME/.gradle`) [...]. 

https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_properties_and_system_properties